examples/private/README.md

```diff
@@ -0,0 +1,9 @@
+# Basic example
```

Comment:
Private example

Also update the content below since its a private only example